server/src/uuid/model/taxonomy_term/messages.rs

```diff
@@ -103,3 +109,35 @@ pub mod taxonomy_term_move_mutation {
         }
     }
 }
+
+pub mod taxonomy_term_create_mutation {
+    use super::*;
+    use crate::instance::Instance;
+    use crate::uuid::Uuid;
+    use crate::vocabulary::model::TaxonomyType;
+
+    #[derive(Deserialize, Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct Payload {
+        pub user_id: i32,
+        pub taxonomy_type: TaxonomyType,
+        pub parent_id: Option<i32>,
+        pub instance: Instance,
```

Comment:
actually not needed since we can get the instance by the parent_id... (or we need a check that instance of parent_id = instance). -> I would delete this parameter.

Reply:
Since root will be done by migration, it is indeed unnecessary. Removing it...